en/techdocs/api/client/hoodie.md

```diff
@@ -29,253 +31,337 @@ This document describes the functionality of the hoodie base object.
 
 
 <a id="baseUrl"></a>
-### hoodie.baseUrl
+### [hoodie.baseUrl](#baseUrl)
 **version:**    *> 0.2.0*
 
-<pre><code>hoodie.baseUrl</code></pre>
+**Reveals Hoodie's base URL**
 
-The **hoodie.baseUrl** property gets automatically set on initialization.
+```javascript
+hoodie.baseUrl
+```
 
+The **hoodie.baseUrl** property is automatically set on initialization, but you can override it when you initialise Hoodie, and provide your own endpoint. This means you can conveniently use the same data store for multiple apps, if you want to.
 
 ##### Example
 
-<pre><code>hoodie = new Hoodie('http://myhoodieapp.com')
-hoodie.baseUrl // 'http://myhoodieapp.com'
+Default initialisation of Hoodie
+
+```javascript
+hoodie = new Hoodie();
+hoodie.baseUrl; // ''
+```
 
-hoodie = new Hoodie()
-hoodie.baseUrl // ''
-</code></pre>
+Specifying your own endpoint
 
+```javascript
+hoodie = new Hoodie('http://myhoodieapp.com');
+hoodie.baseUrl; // 'http://myhoodieapp.com'
+```
 
 <a id="id"></a>
-### id()
+### [id()](#id)
 **version:**      *> 0.2.0*
 
-*Returns a unique, persistent identifier for the current user.*
+**Returns a unique, persistent identifier for the current user.**
 
-<pre><code>hoodie.id();</code></pre>
+```javascript
+hoodie.id();
+```
 
-The first time **hoodie.id()** gets called, a unique
-identifier gets generated for the current user, and persisted
-in the browser's local store. When signing in to an account,
-**hoodie.id()** will return the id of the new user.
-On signout, hoodie.id() gets reset.
+Each Hoodie user is assigned a random, unique, persistent ID on signup. While the user is logged in, **id()** will always return that same ID. When no user is signed in, it will return a different, non-persistent ID for the current anonymous user. Every time a user signs out, the ID for the anonymous user changes.
```

Comment:
>  Each Hoodie user is assigned a random ... ID on signup

This is not correct, the id is assigned the first time you open a hoodie app, and it gets persisted right away. So on page reload, it remains the same. When you sign up for an account, the Hoodie ID remains the same